src/hyperspace/client/richworld.clj

```diff
@@ -4,7 +4,14 @@
   (:use [hyperspace.client particles]
         [hyperspace.library geometry]))
 
-;;; Missles related stuff
+;;; Player-related stuff:
+
+(defn enrich-player

```

Comment:
What do you mean by "enriching" stuff? Is that some kind of commonly known term?

Reply:
The module is called `richworld`. The process of conversion of ordinary world into a rich one I call "enrichment". Consulted with my dictionary.

Reply:
Ok, thx